Patch readme for v0.2.0 (#5)
rancher/opni-metrics-service: README.md

Make sure to update the metric_anomaly_detector.yaml file to point to the correct image path!

#### Import Grafana Dashboard

> 1. Navigate to Grafana and login. For Rancher monitoring user, the username/password is `admin/prom-operator`, otherwise it's likely to be `admin/admin`

Reviewer: ```suggestion
1. Navigate to Grafana and log in. For a Rancher Monitoring user, the default username/password is `admin/prom-operator`. Otherwise, it is likely to be `admin/admin`.
```

> 2. [Add Elasticsearch as a datasource](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/#add-a-data-source), and fillin these fields with following values:

Reviewer: ```suggestion
2. [Add Elasticsearch as a Data Source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/#add-a-data-source), and fill in these fields with following values:
```

```
URL: https://opni-es-client.opni.svc:9200
Basic Auth: enable
Skip TLS Verify: enable
Basic Auth Details:
    User: admin
    Password: <opni-es-password>
Index name: mymetrics
Time field name: timestamp
Version: 7.0+
```

> Then click `Save & Test`, the expected response is `Index OK. Time field name OK.`

Reviewer: ```suggestion
Then click `Save & Test`. The expected response should be `Index OK. Time field name OK.`
```

3. [Import dashboard](https://grafana.com/docs/grafana/latest/dashboards/export-import/#import-dashboard) and upload the json file `grafana-dashboard.json` in this repo.

> A dashboard named `MetricAnomaly` should be ready to go.

Reviewer: ```suggestion
A dashboard named `MetricAnomaly` should now be available.
```
